components/Brand.vue

```diff
@@ -1,20 +1,8 @@
 <template>
 	<div class="brand">
-		<img src="~/static/img/We Count logos_colour and bw-01.png">
-		<h1> We Count </h1>
+		<nuxt-link to="/">
+			<img src="~/static/img/We Count logos_colour and bw-01.png" alt="Website Logo">
```

Comment:
It's good to be a bit more specific -- an alt tag should describe the image. I would change this to "Logo for Project We Count"

Reply:
Okay, I have made the change